README.md

```diff
@@ -28,7 +28,9 @@ This project is organized in a monorepo structure and contains the following wor
 
 ## Scripts
 
-- **w** - core script that allows you to run all workspace scripts from the root folder. To use it you need to follow this script with the workspace name and the script you want to run inside it. For example, running `yarn w website dev` will run the `dev` script inside the `website` workspace.
+- **w** - core script that allows you to run all workspace scripts from the root folder. To use it you need to follow
+  this script with the workspace name and the script you want to run inside it. For example, running
+  `yarn w website dev` will run the `dev` script inside the `website` workspace.
```

Comment:
Why you moved those pieces of text to new lines?

Reply:
Prettier 😫